shared/src/vyxal/StringHelpers.scala

```diff
@@ -1,5 +1,7 @@
 package vyxal
 
+import vyxal.*
```

Comment:
Why the import?

Reply:
For `VList`, right? Or is that imported somewhere else?

Reply:
Scala is like Java in that it's already imported you don't need to manually do so unless you're doing something strange with packaging

Reply:
Oh cool